pkg/metrics/metrics.go

```diff
@@ -355,6 +355,11 @@ var (
 	// Identity is the number of identities currently in use on the node by type
 	Identity = NoOpGaugeVec
 
+	// IdentityLabelSources is the number of identities in use on the node with
+	// have a particular label source. Note that an identity may contain labels
+	// from multiple sources and thus might be counted in multiple buckets
+	IdentityLabelSources = NoOpGaugeVec
```

Comment:
Would it be possible to add things using cell.Metric instead of adding more to pkg/metrics/?